src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/ToolConsole.cs

```diff
@@ -183,7 +157,7 @@ internal static void WriteXmlSerializerTypeGenerationHelp()
                 HelpCategory helpCategory = new HelpCategory(SR.Format(SR.HelpXmlSerializerTypeGenerationCategory));
 
                 helpCategory.Description = SR.Format(SR.HelpXmlSerializerTypeGenerationDescription, ThisAssembly.Title);
-                helpCategory.Syntax = SR.Format(SR.HelpXmlSerializerTypeGenerationSyntax, ThisAssembly.Title, Options.Abbr.Target, Options.Targets.XmlSerializer, SR.Format(SR.HelpInputAssemblyPath));
+                helpCategory.Syntax = SR.Format(SR.HelpXmlSerializerTypeGenerationSyntax, ThisAssembly.Title, SR.Format(SR.HelpInputAssemblyPath));
 
```

Comment:
You don't need to wrap SR.HelpInputAssemblyPath with SR.Format. SR.Format does the equivalent of string.Format when you have parameters, but no parameters are being passed. Just use SR.HelpInputAssemblyPath without the SR.Format wrapping call.